src/pages/LoginPage.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser, faKey } from '@fortawesome/free-solid-svg-icons';
+
+import '../assets/styles/pages/LoginPage.css';
+
+function LoginPage(){
+    const [emailErrorVisibility, setEmailErrorVisibility] = useState('hidden');
+    const [passwordErrorVisibility, setPasswordErrorVisibility] = useState('hidden');
+
+    const [emailErrorMessage, setEmailErrorMessage] = useState('');
+    const [passwordErrorMessage, setPasswordErrorMessage] = useState('');
+
+    const [currentEmail, setCurrentEmail] = useState('');
+    const [currentPassword, setCurrentPassword] = useState('');
+
+    const handleClick = e => {
```

Comment:
Try to make simple this handle submit, **Refactor is the key**

Reply:
Done, I refactored it using the new state